src/main/java/roomescape/RoomescapeController.java

```diff
@@ -31,7 +36,20 @@ public String reservation() {
     @GetMapping("/reservations")
     @ResponseBody
     public ResponseEntity<List<Reservation>> read() {
```

Comment:
read가 어떤 것을 받아오는 메소드인지 확실하게 정의하면 좋을 거 같아요. 또한 현재는 예약 목록을 가져오는 메소드이니, find는 어떨까요? 표현한다면 findAllReservation도 좋을 거 같네요 :D 아래 delete도 그렇답니다!
```suggestion
    public ResponseEntity<List<Reservation>> findReservations() {
```

Reply:
좋은 말씀 감사합니다! 수정했어용
0addd7c8f8233ca8dda6f5acadd070ffc39972c4